### DISCUSSION TITLE
Assign different channels with different URLS?

### DISCUSSION THREAD

**Discussion by AxxPanda** [saleor/saleor#11380] [Q&A]
It would be nice to manage multiple storefronts with the same backend system. For example assigning different URL's to different channels with their own store template. In effect managing multiple sites with same backend and inventory. 

For example
Channel A - www.a.com / with its own storefront template
Channel B - www.bcom / with its own storefront template

**Comment by karolkielecki** [comment]
Hi @AxxPanda, Saleor supports multichannel. You can manage multiple sales channels from a single dashboard. You can find more info in our documentation. 

- https://docs.saleor.io/docs/3.x/developer/channels
- https://docs.saleor.io/docs/3.x/dashboard/channels

If that's not what you're looking for, could you please tell us more details about your use case? 
Cheers,

**Reply by AxxPanda** [reply]
Thanks for the reply Karol, I wanted to ask if it's possible for a channel to have its own unique headless storefront different from other channels and its own url?

**Comment by timuric** [comment]
@AxxPanda Saleor doesn't know about existence of your storefront, you are free to query backend however you want.
[www.a.com](http://www.a.com/) can query `products(channel: "eu")`
[www.b.com](http://www.b.com/) can query `products(channel: "us")`

**Reply by dgeep-web** [reply]
This old comment was very useful to me, but I thought I might ask for some more detail if possible, from anyone willing to reply. The channel model still confuses me sometimes, since I am planning a simple B2B structure to my store. I understand the approach suggested above: Two different domains each with their own store setup to use their respective channels in every query.
But would this also work?:
www.a.com/b2b-cust-A/ can query `products(channel: "cust-A")`
www.a.com/b2b-cust-B/ can query `products(channel: "cust-B")`

**Reply by korycins** [reply]
This will work. The storefront placed on `www.a.com` will communicate with your Saleor `<your-saleor-url>`.  Then by using `b2b-cust-A` and `b2b-cust-B` you can control the value of the `channel`  passed to `<your-saleor-url>` API.
